pkg/pipelines/pipelines.go

```diff
@@ -398,3 +400,62 @@ func GetLatestPipelinerun(c *clients.Clients, namespace string) (string, error)
 	return prs.Items[0].Name, nil
 
 }
+
+func CheckLogVersion(c *clients.Clients, binary, namespace string) {
+	prname, err := GetLatestPipelinerun(store.Clients(), store.Namespace())
+	if err != nil {
+		testsuit.T.Fail(fmt.Errorf("Failed to get PipelineRun: %v", err))
+		return
+	}
+
+	switch binary {
+	case "tkn-pac":
+		expectedVersion := os.Getenv("PAC_VERSION")
+		// Get PipelineRun logs
+		logsBuffer, err := GetPipelinerunLogs(c, prname, namespace)
+		if err != nil {
+			testsuit.T.Fail(fmt.Errorf("Failed to get PipelineRun logs: %v", err))
+			return
+		}
+		// Convert logs to string and extract tkn-pac version using regex
+		logs := logsBuffer.String()
+		re := regexp.MustCompile(`\b(\d+\.\d+\.\d+)\b`)
+		logVersion := re.FindAllString(logs, -1)
+		parts := strings.Split(logVersion[0], ".")
+		if len(parts) < 2 {
+			testsuit.T.Fail(fmt.Errorf("Invalid tkn-pac Version: %s", logVersion[0]))
+			return
+		}
+		logVersion[0] = strings.Join(parts[:2], ".")
+
+		// Compare log version with expected version
+		if logVersion[0] != expectedVersion {
+			testsuit.T.Fail(fmt.Errorf("tkn-pac has an unexpected version: %s, Expected %s", logVersion[0], expectedVersion))
+		}
```

Comment:
Why such a complicated implementation? In other functions we just check that string `contains` expected version. Which additional cases are covered by you implementation?

Reply:
I don't understand... `tkn-pac version` prints something like `0.33.0` and `PAC_VERSION` will be `0.33`. You can easily do substring search.

`tkn version` prints something like `Client version: 0.40.0` and env variable will contain `0.40` so again you can easily do substring search.

Reply:
Done